examples/lock-app/silabs/include/LockManager.h

```diff
@@ -244,17 +270,44 @@ class LockManager
     static void ActuatorMovementTimerEventHandler(AppEvent * aEvent);
 
     osTimerId_t mLockTimer;
-    EmberAfPluginDoorLockUserInfo mLockUsers[kMaxUsers];
-    EmberAfPluginDoorLockCredentialInfo mLockCredentials[kNumCredentialTypes][kMaxCredentials];
-    WeekDaysScheduleInfo mWeekdaySchedule[kMaxUsers][kMaxWeekdaySchedulesPerUser];
-    YearDayScheduleInfo mYeardaySchedule[kMaxUsers][kMaxYeardaySchedulesPerUser];
-    HolidayScheduleInfo mHolidaySchedule[kMaxHolidaySchedules];
-
-    char mUserNames[MATTER_ARRAY_SIZE(mLockUsers)][DOOR_LOCK_MAX_USER_NAME_SIZE];
-    uint8_t mCredentialData[kNumCredentialTypes][kMaxCredentials][kMaxCredentialSize];
-    CredentialStruct mCredentials[kMaxUsers][kMaxCredentials];
 
     EFR32DoorLock::LockInitParams::LockParam LockParams;
+
+    // Stores LockUserInfo corresponding to a user index
+    static StorageKeyName LockUserEndpoint(uint16_t userIndex, chip::EndpointId endpoint)
+    {
+        return StorageKeyName::Formatted("g/lu/%x/e/%x", userIndex, endpoint);
```

Comment:
For all of these, having the endpoint be before other parts of the key might help with pruning things, but not a big deal either way.  Conceptually user index is scoped to endpoint; endpoint is not scoped to user index.

Reply:
Made this change.